tsconfig.json

```diff
@@ -1,18 +1,12 @@
 {
 	"compilerOptions": {
-		"target": "ES2021",
-		"lib": ["ESNext"],
-		"skipLibCheck": true,
+		"target": "ES2022",
 		"strict": true,
-		"alwaysStrict": true,
-		"forceConsistentCasingInFileNames": true,
 		"esModuleInterop": true,
-		"module": "commonjs",
-		"moduleResolution": "node",
-		"isolatedModules": true,
 		"rootDir": "./",
+		"module": "ES2022",
+		"moduleResolution": "Node",
```

Comment:
`NodeNext` is the recommendation for modern projects, which I assume this is.[^1]

[^1]: https://www.typescriptlang.org/tsconfig#module

Reply:
Thanks for the tip!